platform/quality-assurance/auth-session-loadtest.md

```diff
@@ -0,0 +1,36 @@
+# Using authenticated sessions for load testing
+
+## Things you need
+1. [SOCKS proxy access](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/internal-tools/)
```

Comment:
links for #1 and #2 are out of date, see
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/onboarding/request-access-to-tools.md

Reply:
Fixed